data-and-functions-1/src/functions/getMostExpensiveProduct.js

```diff
@@ -0,0 +1,17 @@
+import DATA from '../DATA'
+
+const getMostExpensiveProduct = (data) => {
+  if (data == null) {
+    return null
+  }
+  let mostExpensiveProduct = 0
```

Comment:
There are many ways to solve this one. You need to declare a variable like you did, but let's not initialize it: `let mostExpensiveProduct`